pkg/driver/gcp-pd/gcp_pd.go

```
@@ -0,0 +1,46 @@
+		AssetDir:                        generatedAssetBase,
+		OperatorControllerConfigBuilder: GetGCPPDOperatorControllerConfig,
+		Removable:                       false,
+	}
+}
+
+// GetGCPPDOperatorControllerConfig returns second half of runtime configuration of the CSI driver operator,
+// after a client connection + cluster flavour are established.
+func GetGCPPDOperatorControllerConfig(ctx context.Context, flavour generator.ClusterFlavour, c *clients.Clients) (*config.OperatorControllerConfig, error) {
+	if flavour != generator.FlavourStandalone {
+		klog.Error(nil, "Flavour HyperShift is not supported")
+		return nil, fmt.Errorf("Flavour HyperShift is not supported")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)